src/pages/Product/ProductList/ProductList.js

```diff
@@ -1,7 +1,60 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import ProductNavTab from './ProductNavTab/ProductNavTab';
+import ProductFilterTab from './ProductFilterTab/ProductFilterTab';
+import Product from './Product/Product';
+import '../ProductList/ProductList.scss';
 
 function ProductList() {
-  return <div>ProductList1</div>;
+  const [products, setProducts] = useState([]);
+
+  useEffect(() => {
+    fetch('http://localhost:3000/data/ProductListMock/ProductListMock.json')
```

Comment:
```suggestion
    fetch('/data/ProductListMock/ProductListMock.json')
```
* http://localhost:3000 은 생략 가능합니다. 포트 번호가 달라지면(ex. localhost:3001) 코드 전체를 바꿔야 하기 때문에 수정해주시기 바랍니다. 아래 코드 참고해주세요.
```javascript
const foo = () => {
	// fetch("http://localhost:3000/images/1")
	fetch("/images/1")
}
```